CONTRIBUTING.md

```diff
@@ -62,6 +62,11 @@ make test-integration
 Run e2e testing. During e2e testing, hub and managed cluster share the same Kubernetes cluster.
 ```shell
 export KUBECONFIG=</path/to/kubeconfig>
+
```

Comment:
should we also add go install image-builder here?

Reply:
Of course. 
Added.